check/format-incremental

```diff
@@ -80,25 +80,25 @@ if (( only_changed == 1 )); then
         rev="${base}"
     fi
 
-    # Get the modified, added and moved python files.
+    # Get the modified, added and moved python and notebook files.
     IFS=$'\n' read -r -d '' -a format_files < \
         <(git diff --name-only --diff-filter=MAR "${rev}" -- \
-        '*.py' ':(exclude)*_pb2.py')
+        '*.py' '*.ipynb' ':(exclude)*_pb2.py')
 else
-    echo -e "Formatting all python files." >&2
+    echo -e "Formatting all python and notebook files." >&2
     IFS=$'\n' read -r -d '' -a format_files < \
-        <(git ls-files '*.py' ':(exclude)*_pb2.py')
+        <(git ls-files '*.py' '*.ipynb' ':(exclude)*_pb2.py')
 fi
 
 if (( ${#format_files[@]} == 0 )); then
     echo -e "\033[32mNo files to format.\033[0m"
     exit 0
 fi
 
-# Exclude __init__.py files from isort targets
+# Exclude notebooks and __init__.py files from isort targets
```

Comment:
This says notebooks, but the code below tests against .py files. I'm probably missing something, but flagging it anyway in case it's a genuine discrepancy.

Reply:
Thanks for flagging this.  I have hopefully clarified the comment in 7b907c637e1f3b008309d49bc848b82a960fcfeb.